cmd/platform/deployment-template/show_test.go

```diff
@@ -0,0 +1,66 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package cmddeploymentdemplate
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/elastic/cloud-sdk-go/pkg/multierror"
+	"github.com/elastic/ecctl/cmd/util/testutils"
+)
+
+func Test_showCmd(t *testing.T) {
+	tests := []struct {
+		name string
+		args testutils.Args
+		want testutils.Assertion
+	}{
+		{
+			name: "incorrect amount of arguments returns error",
+			args: testutils.Args{
+				Cmd: showCmd,
+				Args: []string{
+					"show",
+				},
+			},
+			want: testutils.Assertion{
+				Err: errors.New("accepts 1 arg(s), received 0"),
+			},
+		},
+		{
+			name: "unspecified region returns error",
+			args: testutils.Args{
+				Cmd: showCmd,
+				Args: []string{
+					"show", "default", "--format=hi",
+				},
+			},
+			want: testutils.Assertion{
+				Err: multierror.NewPrefixed("invalid deployment template get params",
+					errors.New("region not specified and is required for this operation"),
```

Comment:
@marclop I'm not quite sure why I'm getting an error here 🤔 Looking at the code for the `testutils` package, the region should be filled by default with `ece-region` if I don't specify the region. But when I try adding `"--region=ece-region"` in the args, it returns `unknown flag: --region`. 

I tried looking around but got stumped, got any ideas?

Reply:
I'll dig in see if I can find it

Reply:
I've found the bug and raised https://github.com/elastic/ecctl/pull/336 which should fix this bug.

Reply:
🙏